### PR TITLE
Conduct Attendance, first draft

### PR DESCRIPTION
**Pull request by korzeniwskya**
Included Attendance first draft, contributors names Alex, Aubrey

### CODE REVIEW

**Comment by aubreypc** [comment]
Changed format --- Travis build is now passing.

File: conduct.md [hunk 1]
```diff
@@ -19,6 +17,29 @@ testing fork
...
+website](https://www.gregorykapfhammer.com/teaching/cs203S2019/), contact the
+[instructor](https://www.gregorykapfhammer.com/contact/), or schedule an
+[appointment](https://www.gregorykapfhammer.com/schedule/).
+
+* **Acceptable Attendance (C):**
+  * Attending all lab sessions on time
+  * Missing sessions with consent of the course instructor
+  * Leaving labs early with consent of teammates or the course instructor
+  * Being available on Slack during allotted lab times
+* **Unacceptable Attendance (U):**
```
Comment (quigley-c) [inline conduct.md]: Consider adding a newline before ```**Unacceptable Attendance (U):**``` to remain consistent with other sections.

**Comment by korzeniwskya** [comment]
Attempted to resolve branch error, addition of Gregory M. Kapfhammer before contributors names.

**Comment by shafferz** [comment]
Looks good! If the build issue is fixed, @Michionlion approves the build, and the merge conflicts get resolved, we should be ready to merge!

**Pull request merged**